var/spack/repos/builtin/packages/sprng/package.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Sprng(AutotoolsPackage):
+    """SPRNG: A Scalable Library For Pseudorandom Number Generation
+       Sprng is a distributed process-aware random number generator that
+       avoids correlations in random number sequences across processes.
+    """
+
+    maintainers = ['kayarre']
+
+    homepage = "http://www.sprng.org"
+    url      = "http://www.sprng.org/Version5.0/sprng5.tar.bz2"
+
+    version('5.0',
+            sha256='9172a495472cc24893e7489ce9b5654300dc60cba4430e436ce50d28eb749a66')
+
+    variant('mpi', default=True, description='Enable MPI support')
+    variant('fortran', default=False, description='Enable Fortran support')
+
+    depends_on('mpi+cxx', when='+mpi')
+
+    def configure_args(self):
+        configure_args = []
+        configure_args += self.enable_or_disable('mpi')
+        configure_args += self.enable_or_disable('fortran')
+        # if '+fortran' in self.spec:
+        #     configure_args.append('--with-fortran')
+
+        if '+mpi' in self.spec:
+            # configure_args.append('--with-mpi')
+            if self.spec.satisfies('^openmpi'):
+                configure_args.append('LIBS=-lmpi_cxx')
```

Comment:
Can you try to use:
```python
mpi_link_flags = self.spec['mpi:cxx'].libs.link_flags
configure_args.append('LIBS={0}'.format(link_flags))
```
? That should be more general and work across multiple MPI implementations.

Reply:
Thank you this worked.
```
mpi_link_flags = self.spec['mpi:cxx'].libs.link_flags
configure_args.append('LIBS={0}'.format(mpi_link_flags))
```